tests/kernel/fpu_sharing/float_disable/testcase.yaml

```diff
@@ -6,11 +6,11 @@ tests:
       - CONFIG_DYNAMIC_INTERRUPTS=y
     tags: fpu kernel userspace
   kernel.fpu_sharing.float_disable.riscv32:
-    filter: CONFIG_CPU_HAS_FPU
+    filter: CONFIG_CPU_HAS_FPU and not CONFIG_SMP
```

Comment:
Better to set CONFIG_SMP=n in the configuration so the test still gets coverage.  Or if it's a problem with the test cases being racy and not MP-safe, then you can run SMP=y but with MP_NUM_CPUS=1, or do it on a per-test-case basis using a "1cpu" ztest case.

Or if the problem is that this PR breaks FPU on RISC-V, then we should fix that. :)

Reply:
I'll check those - hopefully it's not the last case =P

Reply:
Checked it - tests use `k_yield()` to sync, so it implicitly requires to be run on single CPU, so I made it explicit.